schemas/JSON/manifests/v1.1.0/manifest.installer.1.1.0.json

```diff
@@ -132,20 +132,56 @@
         }
       }
     },
+    "InstallerReturnCode": {
+      "type": "integer",
+      "not": {
+        "enum": [ 0 ]
+      },
+      "minimum": -2147483648,
+      "maximum": 429496725,
+      "description": "An exit code that can be returned by the installer after execution"
+    },
     "InstallerSuccessCodes": {
       "type": [ "array", "null" ],
       "items": {
-        "type": "integer",
-        "not": {
-          "enum": [ 0 ]
-        },
-        "minimum": -2147483648,
-        "maximum": 4294967295
+        "$ref": "#/definitions/InstallerReturnCode"
       },
       "maxItems": 16,
       "uniqueItems": true,
       "description": "List of additional non-zero installer success exit codes other than known default values by winget"
     },
+    "ExpectedReturnCodes": {
+      "type": [ "array", "null" ],
+      "items": {
+        "type": "object",
+        "properties": {
+          "InstallerReturnCode": {
+            "$ref": "#/definitions/InstallerReturnCode"
+          },
+          "ReturnResponse": {
+            "type": [ "string", "null" ],

```

Comment:
>"null" [](http://example.com/codeflow?start=32&length=6)

This probably can not be null since we don't have defaults for them, let's remove the null from type